AirCasting/SearchAndFollow/CompleteScreen/Views/CompleteScreenViewModel.swift

```diff
@@ -96,7 +134,65 @@ class CompleteScreenViewModel: ObservableObject {
         selectedStream = id
     }
     
-    private static func showStreamName(_ streamName: String) -> String {
+    func xMarkTapped() {
+        isPresented.wrappedValue = false
+    }
+    
+    func dismissView() {
```

Comment:
such method should not be exposed for viewmodels. Public methods should always be a response of some user action and should reflect that in a name. "Dismiss view" is not an action, it's an order, and a view should never tell its viewmodel what to do, only report what happened.

Ofc if this is just an internal method than just put a `private` in front :)